src/typeguard/_transformer.py

```diff
@@ -65,6 +66,8 @@
 from dataclasses import dataclass, field
 from typing import Any, ClassVar, cast, overload
 
+from typing_extensions import override
```

Comment:
Can't have an unconditional import of `typing_extensions` when it's a conditional dependency in `pyproject.toml`. Please remove the `@override` decorators. They're not essential to this change. I'll add them later if I deem them useful.

Reply:
Ignore; I removed it myself.